ipaserver/plugins/cert.py

```diff
@@ -1046,6 +1054,84 @@ def _principal_name_matches_principal(name, principal_obj):
     return principal in principal_obj.get('krbprincipalname', [])
 
 
+def _validate_san_ips(generalnames):
+    """
```

Comment:
Please follow PEP 8 and https://www.python.org/dev/peps/pep-0257/ for doc strings.

Reply:
Will do.  The doc strings in the current PR are just placeholders.